backend/layers/processing/process_validate_atac.py

```diff
@@ -49,28 +49,29 @@ def create_atac_artifact(
         :param file_name: the local file to upload
         :param artifact_type: the type of artifact to upload
         :param dataset_version_id: the dataset version id
-        :param processing_status_key: the key to update the processing status
         :param datasets_bucket: the bucket to upload the dataset to
         :param fragment_artifact_id: the artifact id of the fragment file to be use in the fragment index file.
```

Comment:
```suggestion
        :param fragment_artifact_id: the artifact id of the fragment file, to be used in the fragment index filepath for storage
```

Reply:
reads slightly clearer for me, i figured it out with context but was confused at first at why there were two identifiers